sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/LocalDocumentStorage.java

```diff
@@ -96,6 +98,15 @@
     private static final String BY_PARTITION_AND_DOCUMENT_ID_WHERE_CLAUSE =
             String.format("%s = ? AND %s = ?", PARTITION_COLUMN_NAME, DOCUMENT_ID_COLUMN_NAME);
 
+    /**
+     * String columns can be updated.
+     */
+    private static final String[] MUTABLE_STRING_COLUMNS = new String[]
+            {DOCUMENT_COLUMN_NAME, ETAG_COLUMN_NAME, PENDING_OPERATION_COLUMN_NAME};
+
+    private static final String[] MUTABLE_LONG_COLUMNS = new String[]
```

Comment:
As in long columns cannot be updated?

Reply:
ah they can be updated... 🤔 is it a bad naming problem?

Reply:
this one does have the comment

Reply:
I guess you miss a `not` here 😅 done~